packages/web-twig/src/Resources/components/Tooltip/Tooltip.twig

```diff
@@ -3,6 +3,13 @@
 {%- set _closeLabel = props.closeLabel | default('Close') -%}
 {%- set _id = props.id | default(null) -%}
 {%- set _isDismissible = props.isDismissible | default(false) -%}
+{%- set _enableControlledPlacement = props.enableControlledPlacement | default(false) -%}
+{%- set _enableFlipping = props.enableFlipping ?? true -%}
+{%- set _enableFlippingCrossAxis = props.enableFlippingCrossAxis ?? true -%}
+{%- set _enableShifting = props.enableShifting ?? true -%}
+{%- set _enableSizing = props.placement ?? true -%}
+{%- set _flipFallbackAxisSideDirection = props.flipFallbackAxisSideDirection | default('none') -%}
+{%- set _flipFallbackPlacements = props.flipFallbackPlacements | default(null) -%}
```

Comment:
Ok, I have expanded the `render with all props` test case. I think that this test case covers what you have written above because all data attributes are rendered. So do you think that a separate test case is still needed?